src/components/Applicant/ApplicantCard.tsx

```diff
@@ -14,15 +16,19 @@ export interface IApplicantCard {
     cvLink: string,
 }
 
+const Article = styled.article`
```

Comment:
Not sure how I feel about naming the styled component the same as its semantic counterpart

Reply:
I will change it to Card, that is probably a better description of it